bin/nodist

```diff
@@ -1,10 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
-if [ "$1" = "env" -o "$1" = "use" ]; then
-    \nodist + $2
-    if [ $? -eq 0 ]; then
-        export NODIST_VERSION=$2
-    fi
-else
-    \nodist $*
-fi
\ No newline at end of file
+DIR=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+"$DIR/../node.exe" "$DIR/../cli" "$@"
```

Comment:
It was reported that $DIR can be empty (probably because `$0` doesn't contain that path in cygwin), which breaks this script.